tkn/tkn.yaml

```diff
@@ -0,0 +1,20 @@
+---
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: tkn
+spec:
+  inputs:
+    params:
+      - name: tkn-image
+        description: tkn CLI container image to run this task
+        default: gcr.io/tekton-releases/dogfooding/tkn
```

Comment:
We should tag a stable release for the default.

Reply:
AFAIK this is the best we have right now. Once https://github.com/tektoncd/cli/issues/497 is resolved, we can update this URL.

Reply:
Yes, my apologies. I opened that issue after seeing this pr and became aware from it that we need an official image.